src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: fit-content;
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
```

Comment:
The selector `&--4 &__star:nth-child(-n + 4)` might be confusing. Ensure that the BEM naming convention is followed correctly. Typically, modifiers like `--4` should be used to modify the block or element they are attached to, not combined with other elements or blocks. Consider revising this selector to better align with BEM principles.